Cargo.toml

```diff
@@ -8,9 +8,26 @@ repository = "https://github.com/khorolets/near-ledger-rs/"
 license = "GPL-3.0-or-later"
 keywords = ["ledger", "nearprotocol"]
 
+[[example]]
+name = "get_version"
+
+[[example]]
+name = "get_public_key"
+
+[[example]]
+name = "sign_transaction"
+
 [dependencies]
 ed25519-dalek = { version = "1" }
 ledger-transport = "0.10.0"
 ledger-transport-hid = "0.10.0"
 ledger-apdu = "0.10.0"
 slip10 = "0.4.3"
+log = "0.4.20"
+hex = "0.4.3"
```

Comment:
Are log and hex crates already part of the dependency tree? If not, I would consider making them hidden behind a feature flag

Reply:
these are also indirect dependencies of `ledger-transport-hid`